### PR TITLE
Add func add_panic_device

### PR DESCRIPTION
**Pull request by kylazhang**
Signed-off-by: Kylazhang <weizhan@redhat.com>

### CODE REVIEW

File: test-providers.d/io-github-autotest-libvirt.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [provider]
-uri: https://github.com/autotest/tp-libvirt.git
+uri: file:///root/tp-libvirt/
```
Comment (sathnaga) [inline test-providers.d/io-github-autotest-libvirt.ini]: unness change, pls remove it.

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -2232,6 +2233,24 @@ def create_channel_xml(params, alias=False, address=False):
...
+def add_panic_device(vm_name, model='isa', addr_type='isa', addr_iobase='0x505'):
+    """
+    Create panic device xml
+
+    :param vm_name: name of vm
+    :param model: panic model
+    :param addr_type: address type
+    :param addr_iobase: address iobase
+    """
+    vmxml = vm_xml.VMXML.new_from_dumpxml(vm_name)
```
Comment (sathnaga) [inline virttest/utils_test/libvirt.py]: what will happen if there is a panic device already present in xml, do you want to handle that logic aswell?

File: test-providers.d/io-github-autotest-libvirt.ini (hunk 1 above)
Comment (kylazhang) [reply test-providers.d/io-github-autotest-libvirt.ini]: fixed, thanks

File: virttest/utils_test/libvirt.py (hunk 2 above)
Comment (kylazhang) [reply virttest/utils_test/libvirt.py]: Fixed. Thanks

**Comment by kylazhang** [comment]
@balamuruhans @sathnaga please help to try it with https://github.com/autotest/tp-libvirt/pull/2280 on your env to make sure it can succeed, thanks

**Pull request merged**